build.sbt

```diff
@@ -33,7 +33,6 @@ lazy val mimaSettings = mimaDefaultSettings ++ Seq(
     ProblemFilters.exclude[DirectMissingMethodProblem]("play.libs.ws.ahc.StandaloneAhcWSResponse.getBodyAsSource"),
     ProblemFilters.exclude[MissingClassProblem]("play.api.libs.ws.package$"),
     ProblemFilters.exclude[MissingClassProblem]("play.api.libs.ws.package"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("play.api.libs.ws.ahc.DefaultStreamedAsyncHandler.this")
```

Comment:
If we reset the `previousVersion` to `1.1.13` why not remove all filters?

Still, why reset the `previousVersion` to `1.1.13` and not `1.0.0` (base of the series)?

Reply:
> If we reset the `previousVersion` to `1.1.13` why not remove all filters?

Good thinking.

> Still, why reset the `previousVersion` to `1.1.13` and not `1.0.0` (base of the series)?

To safeguard from breaking binary compatibility between the versions in between.  For instance, 1.1.14 shouldn't remove API published in 1.1.13.

The exception to the rule is if you catch a mistake early and declare the release dead-on-arrival - then you can pretend than release never existed and don't need to be backwards-compatible with it.